src/Elastic.Apm/Model/Payload/Error.cs

```diff
@@ -18,6 +18,7 @@ public ErrorDetail()
 				Id = Guid.NewGuid();
 			}
 
+			[JsonProperty]
```

Comment:
Hahh, this is a very good catch! (and btw. totally my fault 😊 ). 

The strategy around this is to make properties that we want to send to the server within `internal` types to `public`. So, could you please remove `[JsonProperty]` and make it `public`. This is what we do in `Transaction` and `Span`. `Error` itself is still `internal`, so the property won't be visible to the outside world anyway, but Json.NET will serialize it that way. We consider these things `public` within the APM solution, so it's public to the server. 

Thx, this is a really good catch!

Reply:
Hi @gregkalapos , done now it should be public, thanks